webapp/channels/src/components/suggestion/at_mention_provider/at_mention_provider.tsx

```diff
@@ -181,8 +229,8 @@ export default class AtMentionProvider extends Provider {
         }
 
         const remoteMembers = (this.data.users || []).
```

Comment:
There's an error for `this.data.users`
```
Property 'users' does not exist on type 'never'.
```

Reply:
this is beacuse, on top of the file you are setting the type of `data` to `null`. Please try to find the correct type for `data`